docs/proposals/backup/backup.md

```diff
@@ -0,0 +1,735 @@
+	// If multiple provided they will be joined by the OR operator.
+	// LabelSelector as well as OrLabelSelectors cannot co-exist in backup request, only one of them can be used.
+	// +optional
+	// +nullable
+	OrLabelSelectors []*metav1.LabelSelector `json:"orLabelSelectors,omitempty"`
+}
+```
+
+
+The Velero support **Global Scope Resource Filtering**: When specific ns-scope resources are designated, if their descriptions involve cluster-scope resources, 
+Velero will automatically back up these essential resources without requiring users to configure them separately. However, users can still manually configure the global scope resource filtering.
+
+
+##### Unified Restore API
+
+Below is the initial design for the Unified Restore API:
+
+```console
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Namespaced,categories=kurator-dev
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Phase",type="string",JSONPath=".status.phase",description="Phase of the Restore"
+
+// Restore is the schema for the Restore's API.
+type Restore struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              RestoreSpec   `json:"spec,omitempty"`
+	Status            RestoreStatus `json:"status,omitempty"`
+}
+
+type RestoreSpec struct {
+	// BackupName specifies the backup on which this restore operation is based.
+	BackupName string `json:"backupName"`
+
+	// Destination indicates the clusters where restore should be performed.
+	// +optional
+	Destination *Destination `json:"destination,omitempty"`
+
+	// Policy defines the customization rules for the restore.
+	// If null, the backup will be fully restored using default settings.
+	// +optional
+	Policy *RestorePolicy `json:"policy,omitempty"`
+}
+
+// Note: partly copied from https://github.com/vmware-tanzu/velero/pkg/apis/restore_types.go
+// RestorePolicy defines the specification for a Velero restore.
+type RestorePolicy struct {
+	// ResourceFilter is the filter for the resources to be restored.
+	// If not set, all resources from the backup will be restored.
+	// +optional
+	ResourceFilter *ResourceFilter `json:"resourceFilter,omitempty"`
+
+	// NamespaceMapping is a map of source namespace names
+	// to target namespace names to restore into.
+	// Any source namespaces not included in the map will be restored into
+	// namespaces of the same name.
+	// +optional
+	NamespaceMapping map[string]string `json:"namespaceMapping,omitempty"`
+
+	// RestoreStatus specifies which resources we should restore the status field.
+	// If nil, no objects are included.
+	// +optional
+	// +nullable
+	RestoreStatus *RestoreStatusSpec `json:"restoreStatus,omitempty"`
+
+	// PreserveNodePorts specifies whether to restore old nodePorts from backup.
+	// +optional
+	// +nullable
+	PreserveNodePorts *bool `json:"preserveNodePorts,omitempty"`
```

Comment:
what's the default value mean

Reply:
if using the default configuration, the nodePorts will not be preserved or explicitly configured. This means that during the restore process, the nodePorts may be changed or removed. The restored Service will not retain the nodePorts from the previous backup but will instead use the default auto-assigned nodePorts.

Reply:
we should add that. IIUC, it should be documeneted Like: If not specified, default to false

Reply:
If default to false, i think we can use value instead of pointer, right?

Reply:
> we should add that. IIUC, it should be documeneted Like: If not specified, default to false

you are  right